image-prod.yaml

```diff
@@ -0,0 +1,55 @@
+- schema_version: 1
+  name: "builder"
+  description: "Golang builder image"
+  version: ""
+  from: "registry.redhat.io/ubi8/go-toolset:1.17.7"
+  modules:
+    repositories:
```

Comment:
you could use something similar:

```
  modules:
    repositories:
      - path: modules
    install:
      - name: rhpam.builder.prod
```

Reply:
@spolti in this case the _install_ and _module.yaml_ files will be on modules folder root? Why don't keep it in modules/builder-prod path?

Reply:
no, see the `path: modules` property.

Reply:
@spolti ok! changed to the suggested.